workbox-cli-config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  "globDirectory": "./_site/",
```

Comment:
Unfortunately this is not going to work. `_site` is a special folder generated by Jekyll, where it precompiles all of the pages. Unless you have Jekyll installed locally and run `jekyll build/serve/etc.` it will not be created. We run our site updates through the script executed on our CI server. Our CI server doesn't have Jekyll installed and because of that, this folder isn't going to be available. Using `/` isn't going to work either, because in the root, all of the files are partials, without headers and footers. They are also `md` files, and not `html` files.
In short, I really don't have a suggestion as to how to get this to work. The only thing I can think of, is installing Jekyll on our CI server, but I really don't want to do that. Jekyll is very brittle and breaks constantly because of the new versions released. Our CI server is also pretty small (hardware wise), and I really don't want to install Ruby on it, and slow it down even more.
@eslint/eslint-team anyone has any ideas on how we can do this?